app/features/quizzify/tools.py

```diff
@@ -470,7 +420,7 @@ def __init__(self, loader=None, splitter=None, vectorstore_class=None, embedding
             "loader": URLLoader(verbose = verbose), # Creates instance on call with verbosity
             "splitter": RecursiveCharacterTextSplitter(chunk_size=1000, chunk_overlap=100),
             "vectorstore_class": Chroma,
-            "embedding_model": VertexAIEmbeddings(model='textembedding-gecko')
+            "embedding_model": VertexAIEmbeddings(model_name='textembedding-gecko')
```

Comment:
I believe the main branches are using the new langchain classes for Google with the library `langchain-google-genai` whose classes are called `GoogleGenerativeAI`